apollo-router/src/plugins/telemetry/apollo.rs

```diff
@@ -142,13 +142,17 @@ pub(crate) struct ErrorConfiguration {
     pub(crate) send: bool,
     /// Redact subgraph errors to Apollo Studio
     pub(crate) redact: bool,
+    /// Allows additional dimension `extensions.code` to be sent with errors
+    /// even when `redact` is set to `true`.  Has no effect when `redact` is false.
+    pub(crate) redaction_policy: ErrorRedactionPolicy,
 }
 
 impl Default for ErrorConfiguration {
     fn default() -> Self {
         Self {
             send: true,
             redact: true,
+            redaction_policy: ErrorRedactionPolicy::Strict,
```

Comment:
When we mark the config as non-experimental, should we also update this default to be extended?

Reply:
I'm on the fence here - I think it would ideally be on by default, but I'd love to confirm that with someone on the Router side.  Will ask them for their thoughts before shipping the non-experimental flag.

Reply:
I think right now our policy in the router would probably to keep Strict as default to be more conservative by default.
Also I would suggest to use:
```suggestion
            redaction_policy: ErrorRedactionPolicy::default(),
```

It would be easier for us to change the default behavior in the future by just changing the default impl of `ErrorRedactionPolicy`